### PR TITLE
Single Page Containing Navbar, About Me, Portfolio & Contact Section

### PR DESCRIPTION
**Pull request by MattW-R**
Structured & styled elements.
Accessibility action taken: AAA colour differences on viewing, hovering, focus & selection. Tab direction around page.
Had too much fun with CSS transition implementation everywhere.
Responsiveness to viewport resolution & mobiles added.
Extra settings for browser compatibility implemented.
Refactored & tidied code at the end.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			</div>
+		</main>
+		<footer>
+			<a class="anchor-target" id="contact"></a>
+			<h1 tabindex="5">contact</h1>
+			<div>
+				<div>
+					<a href="mailto:Matthew.wr89@gmail.com" tabindex="5">email<span>: Matthew.wr89@gmail.com</span></a>
+					<a href="https://github.com/MattW-R" tabindex="5">github<span>: MattW-R</span></a>
+					<a href="" tabindex="5">LinkedIn<span>: Matthew W-R</span></a>
```
Comment (charliecog) [inline index.html:83]: awaiting url

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,339 @@
+/*-------------------------------------------------------------------
+						MACRO STYLING
+-------------------------------------------------------------------*/
+
+* {
+	box-sizing: border-box;
+}
+
+/* Don't fear I have added in my own tab focus styling */
```
Comment (charliecog) [inline styles.css]: thanks for the message but please remove the comment

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+}
+
+canvas {
+	image-rendering: crisp-edges;
+}
+
+body {
+	color: #FE75FE;
+	margin-top: 60px;
+	background-image: url(images/BG.png);
```
Comment (charliecog) [inline styles.css]: missing " " around the image url

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+}
+
+footer h1 {
+	background-color: transparent;
+	transition: border-color 200ms ease-in 100ms;
+	border: none;
+	border-bottom: 3px solid transparent;
+	border-radius: 0;
+	display: inline-block;
+	color: white;
```
Comment (charliecog) [inline styles.css]: use a hex code for consistency

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+	}
+}
+
+/*-----------------------------------------------------------------
+						WEBKIT ONLY
+-----------------------------------------------------------------*/
+
+@media screen and (-webkit-min-device-pixel-ratio:0) {  
+	body {
+		background-image: url(images/BGx10.png);
```
Comment (charliecog) [inline styles.css]: missing " "

**Review by charliecog — COMMENTED** [review]
Looks great, only a couple of minimal things to tweak and you will be good to go

File: index.html (hunk 1 above)
Comment (MattW-R) [reply index.html:83]: Need to re-setup LinkedIn, its currently dormant. Was waiting till you talk about it later.

Comment (charliecog) [reply index.html:83]: Sounds good

**Review by charliecog — APPROVED** [review]
Well done!

**Pull request merged**